packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@reatom/core",
   "private": false,
-  "version": "1.0.0",
+  "version": "1.0.2-alpha.6",
```

Comment:
`1.1.0`

Reply:
I will update this PR after publish 1.1.0